crypto/bls12381_utils.c

```diff
@@ -580,33 +570,38 @@ void ep_sum_vector(ep_t jointx, ep_st* x, const int len) {
 }
 
 // Computes the sum of the signatures (G1 elements) flattened in a single sigs array
-// and store the sum bytes in dest.
+// and writes the sum (G1 element) as bytes in dest.
 // The function assumes sigs is correctly allocated with regards to len.
 int ep_sum_vector_byte(byte* dest, const byte* sigs_bytes, const int len) {
+    int error;
+
     // temp variables
     ep_t acc;        
     ep_new(acc);
     ep_set_infty(acc);
     ep_st* sigs = (ep_st*) malloc(len * sizeof(ep_st));
+    for (int i=0; i < len; i++) ep_new(sigs[i]);
```

Comment:
nit: wrap for body in `{}`

Reply:
a very good suggestion: would do a mass-rewrite in a separate diff to programmatically apply this.  Not forcing `{}` on lone ifs and fors is a disaster waiting to happen.

Reply:
I had a quick look at recommended coding styles: 
- [GNU conventions](https://www.gnu.org/prep/standards/html_node/Syntactic-Conventions.html#Syntactic-Conventions) seems to allow not including `{}` in some cases. 
- [Indian hill](http://www.literateprogramming.com/indhill-cstyle.pdf) coding standard also seems to include one line statements without braces. 

Happy to hear your thoughts on why you prefer to enforce braces.

Reply:
The main reasons for using braces are usually:
* less error prone: although quite small, but there is still non-zero probability of accidentally trying to add a line to a single-line if or for.  If we can avoid that with a clang-format rule -- imho we should do that.
* diffs are cleaner: if you need to add code to an existing single line diff you don't bloat the diff.

Also, given that our primary language (go) forces brackets, so forcing them here should make code more familiar.

PS. Kernel style guides (both linux and BSD's `style(9)` do allow single line control statements, but here I think it is bad to rely on them, since their coding styles were invented way before auto-code formatting with `go fmt`/`python-black`/`clang-format` became mainstream.)